crypto/bls12381/bls12381.go

```diff
@@ -8,8 +8,6 @@ import (
 	"errors"
```

Comment:
Getting in line 1: 

directive `//nolint: gosec` is unused for linter "gosec" (nolintlint)

Reply:
done